Automattic/jetpack
# Add deactivation survey to Jetpack Boost

This PR adds a deactivation feedback intercept when Jetpack Boost is deactivated which sends the person to a Crowdsignal survey to collect results.

#### Changes proposed in this Pull Request:
* Loads in a new template file on deactivation of Boost

#### Other information:


#### Jetpack product discussion
Discussion thread `pc9hqz-WO` here.

#### Does this pull request change what data or activity we track or use?
No but we are asking for active consent for feedback. 

#### Testing instructions:
- Switch to this branch
- Activate Jetpack Boost
- Head to the Plugins page
- Deactivate Jetpack Boost 
- The feedback intercept should appear
- Following the link takes you to a Crowdsignal form
- The form only allows one response per computer

<img width="1440" alt="Screenshot 2022-09-13 at 13 40 52" src="https://user-images.githubusercontent.com/4077604/189903571-7482604b-5194-4767-8479-591782d3fac5.png">

![screencapture-surveys-jetpack-me-jetpack-boost-deactivation-feedback-2022-09-13-13_47_37](https://user-images.githubusercontent.com/4077604/189904969-daf91436-8716-4878-abda-d68e4799ef4e.png)
